docs/2026-06-25_aws_gateway_benchmark/translation/capture.py

```
@@ -0,0 +1,218 @@
+    try:
+        resp = urllib.request.urlopen(urllib.request.Request(url, method="GET"), timeout=timeout)
+        return json.loads(resp.read().decode("utf-8"))
+    except Exception:  # noqa: BLE001
+        return None
+
+
+def mock_reset():
+    try:
+        urllib.request.urlopen(urllib.request.Request(MOCK + "/__reset", data=b"", method="POST"), timeout=5).read()
+    except Exception:  # noqa: BLE001
+        pass
```

Ruff (0.15.18)

[warning] Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)